### PR TITLE
Add initial metrics documentation. Fixes #5

### LINKED ISSUE #5: Write metrics.md
We need some metrics documentation

### PR DESCRIPTION
**Pull request by clouserw**

### CODE REVIEW

File: docs/metrics.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  - Breakdown of positive vs negative responses
+
+Each of these should be available globally, as well as broken down per consumer
+application.
+
+
+## Collection
+This project uses Ping Centre to collect metrics data.  Pings will be sent as
+JSON blobs.  All pings will include the following fields:
+- Current timestamp: string
```
Comment (teonbrooks) [inline docs/metrics.md]: is this the only shared metric across the pings?

File: docs/metrics.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+### A new item is submitted from a consumer
+- ID of the consumer: integer
+- Number of items in message queue: integer
+- Type of item submitted (eg. 'png' or 'jpg'): string
+
+### A worker wakes up or sleeps
+A worker wakes up periodically to process the queue.  When it wakes up it
+selects a portion of the queue to process and it shuts down when it finishes
+processing them.  When the worker wakes up *or* shuts down, it will send:
```
Comment (teonbrooks) [inline docs/metrics.md]: for clarification, a worker can shut down with items in the queue still?

File: docs/metrics.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+- Number of items in the queue: integer
+
+### Worker processes the queue
+For *each* item it processes:
+- Timing (in ms):
+  - To retrieve item from the queue: integer
+  - To send and receive a response from PhotoDNA: integer
+  - To send a response to the consumer's report URL: integer
+- Whether the response was positive or negative: string
```
Comment (teonbrooks) [inline docs/metrics.md]: Does positive and negative refer to whether a response was found?

File: docs/metrics.md (hunk 1 above)
Comment (wresuolc) [reply docs/metrics.md]: it's the only one I could think of

File: docs/metrics.md (hunk 2 above)
Comment (wresuolc) [reply docs/metrics.md]: no, it should finish it's queue

File: docs/metrics.md (hunk 3 above)
Comment (wresuolc) [reply docs/metrics.md]: No, the response is either positive or negative based on what was found.  They should all have responses

Comment (teonbrooks) [reply docs/metrics.md]: oh ok, so +/- refers to sentiment?

**Comment by teonbrooks** [comment]
the metrics look fine and is a good starting point for screenshots as the first consumer

**Pull request merged**